test/parallel/test-module-main-extensions.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+const { execFileSync } = require('child_process');
+
+const node = process.argv[0];
+
+try {
+  execFileSync(node, ['--experimental-modules', 'test/es-module/test-esm-ok']);
+} catch (e) {
+  if (e.toString().match(/Error: Cannot find module/))
+    assert.fail('Expected top-level extension lookup to resolve.');
+}
+assert.ok(true);
```

Comment:
This line isn’t going to do anything, right? Am I missing something?

Reply:
This is just from a habit of ensuring there is at least one assertion in a test. Can remove.